package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.0.6",
   "author": "Jeff Dickey @dickeyxxxx",
   "dependencies": {
+
```

Comment:
no, everything needing to be done is done with CI. Fixpack isn't crucial, I just wanted to get rid of the empty line mostly